go/app/main.go

```diff
@@ -20,22 +26,98 @@ type Response struct {
 	Message string `json:"message"`
 }
 
+type Item struct {
+	Name     string `json:"name"`
+	Category string `json:"category"`
+	Image    string `json:"image"`
+}
+
+var items = make(map[string][]Item)
+
 func root(c echo.Context) error {
 	res := Response{Message: "Hello, world!"}
 	return c.JSON(http.StatusOK, res)
 }
 
+func getFileContent() {
```

Comment:
errorが返されている部分があるので、error返しましょう。errorのprintしている部分はerrorを返すようにしてください。
```suggestion
func getFileContent() error {
```